src/c++/vdo/fake/linux/timer.h

```diff
@@ -1,16 +1,21 @@
 /* SPDX-License-Identifier: GPL-2.0-only */
 /*
- * Copyright 2023 Red Hat
+ * Unit test implementations of linux/timer.h.
  *
- * $Id$
+ * Copyright 2024 Red Hat
  */
 
 #ifndef LINUX_TIMER_H
 #define LINUX_TIMER_H
 
-#include <linux/jiffies.h>
 #include <linux/list.h>
 
+/* Defined in linux/timer_types.h
+ * 
```

Comment:
Line 14 still has a trailing space after the *

Reply:
Fixed.